source/configuration_manual/authentication/passwd.rst

```diff
@@ -17,30 +17,44 @@ disabling it:
 .. code-block:: none
 
   userdb passwd {
-    args = blocking=no
+    userdb_use_worker = no
```

Comment:
No need for userdb_ prefix, just `use_worker` to keep it more consistent with other documentation